src/lib/registration/widgets/package_search.rb

```diff
@@ -193,6 +227,20 @@ def valid_search_text?(text)
         )
         false
       end
+
+      # Returns the header to display in the feedback window while searching for packages
+      #
+      # @return [String]
+      def searching_header
+        _("Contacting the SUSE Customer Center. This may take a few seconds.\n")
```

Comment:
Maybe it could take more than few seconds... :wink: 

```suggestion
        _("Contacting the SUSE Customer Center. This may take some time.\n")
```